<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIFontCreation</class>
 <widget class="QWidget" name="UIFontCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" rowspan="7">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="7">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="preview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>196</width>
        <height>490</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>193</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Font Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="currentFont">
         <font>
          <family>Terminal</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbBold">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbItalic">
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="fontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="sbStartChar">
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>33</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="sbEndChar">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>126</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Start char.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>End char.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>193</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Font Movement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="btRotateCCW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rotate counter-clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/object-rotate-left.png</normaloff>:/Icons/resources/object-rotate-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move up</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/gtk-go-up.png</normaloff>:/Icons/resources/gtk-go-up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btRotateCW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rotate clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/object-rotate-right.png</normaloff>:/Icons/resources/object-rotate-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnLeft">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move left</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/gtk-go-back-ltr.png</normaloff>:/Icons/resources/gtk-go-back-ltr.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnRight">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move right</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/gtk-go-back-rtl.png</normaloff>:/Icons/resources/gtk-go-back-rtl.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnDown">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move down</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/gtk-go-down.png</normaloff>:/Icons/resources/gtk-go-down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnMirrorVer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/object-flip-vertical.png</normaloff>:/Icons/resources/object-flip-vertical.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnMirrorHor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/object-flip-horizontal.png</normaloff>:/Icons/resources/object-flip-horizontal.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnReset">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Reset</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/resources/reload.png</normaloff>:/Icons/resources/reload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>193</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>GLCD Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="glcdWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="glcdHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="glcdColor">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Rot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blau</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weiss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grün</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ZoomFactor</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>188</width>
       <height>43</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnCode">
     <property name="maximumSize">
      <size>
       <width>193</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Generate Code</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="maximumSize">
      <size>
       <width>193</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnAutoArrange">
     <property name="toolTip">
      <string>Auto arrange font (still buggy)</string>
     </property>
     <property name="text">
      <string>Auto Arrange</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>glcdWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateGLCDSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>glcdHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateGLCDSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateGLCDZoom(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontComboBox</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbBold</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbItalic</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUp</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>decOffsetY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDown</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>incOffsetY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLeft</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>decOffsetX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRight</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>incOffsetX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCode</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>showCode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btRotateCCW</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>rotateCCW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btRotateCW</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>rotateCW()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbStartChar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateCharList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbEndChar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>updateCharList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>resetPostion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>glcdColor</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>UIFontCreation</receiver>
   <slot>setGLCDColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAutoArrange</sender>
   <signal>clicked()</signal>
   <receiver>UIFontCreation</receiver>
   <slot>autoArrange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFont()</slot>
  <slot>updateGLCDZoom(double)</slot>
  <slot>updateGLCDSize()</slot>
  <slot>incOffsetX()</slot>
  <slot>decOffsetX()</slot>
  <slot>incOffsetY()</slot>
  <slot>decOffsetY()</slot>
  <slot>showCode()</slot>
  <slot>rotateCCW()</slot>
  <slot>rotateCW()</slot>
  <slot>updateCharList()</slot>
  <slot>resetPostion()</slot>
  <slot>setGLCDColor()</slot>
  <slot>autoArrange()</slot>
 </slots>
</ui>
